src/app/containers/FeatureIndexMain/index.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable jsx-a11y/aria-role */
+import React from 'react';
+import path from 'ramda/src/path';
+import { frontPageDataPropTypes } from '#models/propTypes/frontPage';
+import MetadataContainer from '../Metadata';
+import LinkedData from '../LinkedData';
+import ATIAnalytics from '../ATIAnalytics';
+import ChartbeatAnalytics from '../ChartbeatAnalytics';
+import StoryPromoGroups from '../StoryPromoGroups';
+
+const FeatureIndexMain = ({ featureIndexData }) => {
+  const title = path(['metadata', 'title'], featureIndexData);
+  const groups = path(['content', 'groups'], featureIndexData);
+  const lang = path(['metadata', 'language'], featureIndexData);
+  const description = path(['metadata', 'summary'], featureIndexData);
+  const seoTitle = path(['promo', 'name'], featureIndexData);
+
+  return (
+    <>
+      <ATIAnalytics data={featureIndexData} />
+      <ChartbeatAnalytics data={featureIndexData} />
+      <MetadataContainer
+        title={title}
+        lang={lang}
+        description={description}
+        openGraphType="website"
+      />
+      <LinkedData type="WebPage" seoTitle={seoTitle} />
+      <main role="main">
+        <h1>{title}</h1>
```

Comment:
We will need to add `id="content" tabIndex="-1"` to this `h1` so we can jump to it when clicking on the `Skip to content` link.

Reply:
Very good point! We have this ticket: https://github.com/bbc/simorgh/issues/4162 to create/use a title component here, I just put this in as a placeholder - sorry for the confusion! Although that's another a good point, if displaying the title is a seperate ticket, what's the condition of satisfaction for this ticket? Keeping a hidden h1 for now? I'll get clarification.

Reply:
Good point. I forgot about the other ticket when reviewing this, sorry! Perhaps, you could leave it like this in this PR, and deal with it in another?

Reply:
Just spoke to Amy, going to include the title as part of this PR :+1: